examples/ct/handlers.go

```diff
@@ -869,9 +872,12 @@ func marshalGetEntriesResponse(c LogContext, rpcResponse *trillian.GetLeavesByIn
 		// return the data if it fails to deserialize as otherwise the root hash could not
 		// be verified. However this indicates a potentially serious failure in log operation
 		// or data storage that should be investigated.
-		if _, err := ct.ReadMerkleTreeLeaf(bytes.NewBuffer(leaf.LeafValue)); err != nil {
+		var treeLeaf ct.MerkleTreeLeaf
+		if rest, err := tls.Unmarshal(leaf.LeafValue, &treeLeaf); err != nil {
 			// TODO(Martin2112): Hook this up to monitoring when implemented
-			glog.Warningf("%sFailed to deserialize merkle leaf from backend: %d", c.logPrefix, leaf.LeafIndex)
+			glog.Warningf("%sFailed to deserialize Merkle leaf from backend: %d", c.logPrefix, leaf.LeafIndex)
```

Comment:
[unrelated] Hmm, I thought these log lines were changed to `"%s: blah"` in a previous CL, did this file get missed or did that not happen at all?